main.tf

```diff
@@ -19,4 +21,16 @@ module "alerts" {
   rules                  = var.alerts.rules
   contact_points         = var.alerts.contact_points
   notifications          = var.alerts.notifications
+
+  depends_on = [module.grafana, module.prometheus]
+}
+
+module "grafana" {
```

Comment:
why no parameters sent to this submodule, same question for prometheus submodule,
sure they have some defaults but me may need to override the defaults,
also cant get relation between grafana/prometheus and alert/application_dashboard modules, if it is supposed to create grafana and dashboard on it, we seems have to create grafana provider inside the module

Reply:
can we have grafana/prometheus enabled/disable option to control if grafana/peometheus should be created in modules call via congis like:
`count = var.grafana.enabled ? 1 : 0` # in grafana module
`count = var.prometheus.enabled ? 1 : 0` # in prometheus module